src/khepri_tx.erl

```diff
@@ -256,6 +258,14 @@ ensure_instruction_is_permitted({deallocate, _}) ->
     ok;
 ensure_instruction_is_permitted({func_info, _, _, _}) ->
     ok;
+ensure_instruction_is_permitted({fconv, _, _}) ->
+    ok;
+ensure_instruction_is_permitted(fclearerror) ->
+    ok;
+ensure_instruction_is_permitted({fcheckerror, _}) ->
+    ok;
+ensure_instruction_is_permitted({fmove, _, _}) ->
+    ok;
```

Comment:
Should I split the commit so the float arithmetic instructions have their own commit and message?

Reply:
Were they required to make the refactored testsuite work?